composer.json

```
@@ -1,8 +1,8 @@
 {
     "name": "api-scout/api-scout",
     "description": "A library with a few tools, to auto document your api",
-    "type": "library",
-    "license": "proprietary",
+    "type": "symfony-bundle",
```

I want it to stay as a library as it is not necessary linked to a Symfony project
```suggestion
    "type": "library",
```

---

I don't agree, given the architecture and the features this should be intended to be used with the bundle IMO.